src/corehost/common/pal.windows.cpp

```diff
@@ -519,3 +519,9 @@ bool pal::is_running_in_wow64()
     }
     return (fWow64Process != FALSE);
 }
+
+bool pal::are_paths_equal_with_normalized_casing(const string_t& path1, const string_t& path2)
+{
+    // On Windows, paths are case-insensitive
+    return (strcasecmp(path1.c_str(), path2.c_str()) == 0);
```

Comment:
This is:

https://github.com/dotnet/core-setup/blob/24ba120b0ca1ba60a24a79b7dc95866cd3e9f799/src/corehost/common/pal.h#L125

And I believe [_wcsicmp](https://msdn.microsoft.com/en-us/library/k59z8dwe.aspx) behavior varies by locale.

I think it should be [CompareStringOrdinal](https://docs.microsoft.com/en-us/windows/desktop/api/stringapiset/nf-stringapiset-comparestringordinal) with bIgnoreCase=true.

Since we already seem to be using strcasecmp in other places where it should probably be ordinal-ignore-case, I'd be fine with moving this feedback item to an issue to look at it everywhere in the repo.

Reply:
_wcsicmp uses the default "C locale" which should be sufficient, at least for ASCII. Not sure how Unicode file systems would work, however. Created issue https://github.com/dotnet/core-setup/issues/4459 to track